R/plotDispersion.R

```diff
@@ -19,27 +20,32 @@
 #' @param linefitColor (Default = "yellow")
 #' @param lineFit (Default = NULL) Any type supported by geom_smooth (Loess is
 #'   recommended).
-#' @param lineType Any ggplot2 lineType supported by geom_smooth (Default = 1, solid)
+#' @param lineType Any ggplot2 lineType supported by geom_smooth (Default = solid)
 #' @param lineAlpha Alpha desired for geom_smooth line (Default = 1)
-#' @param rugColor (Default = NULL)  Set to Null to disable rug plots.
-#' @param rugAlpha Transparency for the rug plots (Default = 0.02)
 #' @param ... Extra parameters to pass to edgeR::estimateDisp
 #'
-#' @return A ggplot object
+#' @return canvasxpress or ggplot object based on plotType selection
 #'
 #' @examples
 #' \dontrun{
-#'    myGgplot <- plotDispersion(myDGElist)
-#'    myGgplot <- plotDispersion(myDGEobj)
+#'    # canvasxpress plot
+#'    myCxplot <- plotDispersion(myDGElist)
+#'    myCxplot <- plotDispersion(myDGEobj)
+#'
+#'    # ggplot
+#'    myGgplot <- plotDispersion(myDGElist, plotType = "ggplot")
+#'    myGgplot <- plotDispersion(myDGEobj, plotType = "ggplot")
 #' }
 #'
 #' @importFrom assertthat assert_that
 #' @importFrom edgeR calcNormFactors estimateDisp DGEList
+#' @importFrom canvasXpress canvasXpress
 #'
 #' @export
 plotDispersion <- function(DGEdata,
                            designMatrix,
-                           plotType = "dispersion",
+                           plotType = "canvasXpress",
+                           plotValue = "dispersion",
```

Comment:
Can you rename this to plotCategory from plotValue. Category is closer to the word "type" in meaning than "value". We will anyways run this past Connie and get her confirmation